### PR TITLE
transformed to func comp

### PR DESCRIPTION
**Pull request by dwantula**

### CODE REVIEW

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx [hunk 1]
```diff
@@ -7,58 +7,60 @@ import FavouriteButton from '../../shared/components/FavouriteButton/FavouriteBu
...
+  name: '',
+  abv: '',
+  img: '',
+  ibu: '',
+  isPlaceholder: true,
+})
+
+function BeerOfTheDayComponent() {
+
+  const [beer, setBeer] = useState(createEmptyBeer())
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: You should not create function to initiate state. Just pass initial object here:
```
useState({
  name: '',
  abv: '',
  img: '',
  ibu: '',
  isPlaceholder: true
})

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx [hunk 2]
```diff
@@ -7,58 +7,60 @@ import FavouriteButton from '../../shared/components/FavouriteButton/FavouriteBu
...
+
+  const getRandomBeer = async () => {
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: I am not a fan of naming functions with const and arrow function syntax. I strongly suggest to use function naming.

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx [hunk 3]
```diff
@@ -7,58 +7,60 @@ import FavouriteButton from '../../shared/components/FavouriteButton/FavouriteBu
...
     const beer = await fetchRandomBeer()
-    this.setState({ beer });
+    console.log(beer)
```
Comment (mi-chal) [inline src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: Do not leave console.logs in PRs.

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx (hunk 1 above)
Comment (dwantula) [reply src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: Done!

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx (hunk 2 above)
Comment (dwantula) [reply src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: Done!

File: src/screens/BeerOfTheDay/BeerOfTheDay.jsx (hunk 3 above)
Comment (dwantula) [reply src/screens/BeerOfTheDay/BeerOfTheDay.jsx]: Done!

**Review by mi-chal — APPROVED** [review]
Awesome work! 🚀

**Pull request merged**